src/definitions/behaviors/form.js

```diff
@@ -730,42 +730,53 @@
-                                            if (settings.dateHandling == 'date') {
-                                                values[name] = date;
-                                            } else if (settings.dateHandling == 'input') {
-                                                values[name] = $calendar.calendar('get input date');
-                                            } else if (settings.dateHandling == 'formatter') {
-                                                var type = $calendar.calendar('setting', 'type');
+                                            switch (settings.dateHandling) {
+                                                case 'date': {
+                                                    values[name] = date;
 
-                                                switch (type) {
-                                                    case 'date':
-                                                        values[name] = settings.formatter.date(date);
-
-                                                        break;
-
-                                                    case 'datetime':
-                                                        values[name] = settings.formatter.datetime(date);
-
-                                                        break;
-
-                                                    case 'time':
-                                                        values[name] = settings.formatter.time(date);
-
-                                                        break;
-
-                                                    case 'month':
-                                                        values[name] = settings.formatter.month(date);
-
-                                                        break;
-
-                                                    case 'year':
-                                                        values[name] = settings.formatter.year(date);
-
-                                                        break;
+                                                    break;
+                                                }
+                                                case 'input': {
+                                                    values[name] = $calendar.calendar('get input date');
 
-                                                    default:
-                                                        module.debug('Wrong calendar mode', $calendar, type);
-                                                        values[name] = '';
+                                                    break;
+                                                }
+                                                case 'formatter': {
+                                                    var type = $calendar.calendar('setting', 'type');
+
+                                                    switch (type) {
+                                                        case 'date': {
```

Comment:
I know i repeat myself, but there is no need for the overall applied scoping as values[names] belong to upper scope already

Reply:
see my reply above, a style asserted by CI, it should be fine

Reply:
technically fine, i still dont agree without a need of scoping

Reply:
the need of scoping if not evaluated by eslint for each block, it is only a reason why the rule exists, eslint simply wraps are `case` blocks in brackets